### DISCUSSION TITLE
release

### DISCUSSION THREAD

**Discussion by AndreiZhoraven** [schibsted/jslt#263] [General]
Hey everyone, 

could you tell me when you plan to make the next release 0.1.13 which will include a fix forhttps://github.com/schibsted/jslt/issues/261  ? 

Br,
Andrei

**Comment by larsga** [comment]
Since there's interest I'll be happy to make a new release. I did want to get feedback on the solution before releasing it, so I'll call for that, and if I don't hear anything within a few days then let's release.

**Reply by AndreiZhoraven** [reply]
thanks!

**Comment by larsga** [comment]
Looks like we're OK on this. I'll release as soon as I have time. Before the end of the week for sure.

**Comment by larsga** [comment]
Release 0.1.13 is out now.

My apologies for the delay. I finished a book manuscript on Thursday, and completing it took more of my energy than I had anticipated.

**Reply by AndreiZhoraven** [reply]
Super, thanks a lot!
